pkg/applicationserver/io/web/webhooks.go

```diff
@@ -194,6 +201,11 @@ func (w *webhooks) handleDown(
 			webhandlers.Error(res, req, errWebhookNotFound.New())
 			return
 		}
+		if hook.Paused {
+			logger.Debug("Webhook is paused")
+			res.WriteHeader(http.StatusAccepted)
```

Comment:
So we're basically swallowing the downlink message while the webhook is paused? Why not return an error here? The equivalent to `Unavailable`

Reply:
An error seems a bit off for me, if the user himself sets the option to pause the webhook, isn't it a valid case?

Reply:
Yes but something is off, because the API says it accepted the downlink but in fact it threw it away.

I wouldn't expect this happens a lot, because it's unlikely that downlink get scheduled when there's no uplink sent during a pause. Still, in my opinion it's wrong to throw away data without an error. Status codes are intended for APIs to know what's going on.

So instead of `StatusAccepted` please return `StatusNotAcceptable`.

Reply:
The idea with a 2xx error was to prevent multiple potential retries from the external endpoint. And in the vast majority of cases, users pause webhooks intentionally. But yes, in principle we shouldn't swallow errors. I'm fine with changing this to `StatusNotAcceptable` for now and then we can see how this works in production.

Reply:
No, the status code is not addressed yet. Will change to StatusNotAcceptable.

Reply:
Done.